src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/notifications/EmailTemplates.java

```diff
@@ -54,6 +54,12 @@
     @NotEmpty
     private String offerRejectedByClaimantEmailToDefendant;
 
+    @NotEmpty
+    private String offerCounterSignedEmailByOriginator;
```

Comment:
Change to be `offerCounterSignedEmailToOriginator ` - simply replace `By` with `To`